src/DataProtection/DataProtection/src/Internal/Int7BitEncodingUtils.cs

```diff
@@ -0,0 +1,51 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
I wonder if we can move this to /src/Shared; I've seen this type of migration increasingly (outputcache, hybridcache - although the latter isn't in aspnetcore) - it might make sense to have a single well-trusted version. I'm not absolute about this and "zero change" is a fair outcome, but if we can shape out what it might be, we could also perhaps propose a similar to push the code to dotnet/runtime, because: I've seen this too many times lately, with the spanification of all things